signinapp/templates/event_stats.html.jinja2

```diff
@@ -12,6 +18,41 @@
       {%- endif -%}
       <p>Location: {{event.location}}</p>
       <p>{{event.start}} - {{event.end}}</p>
+      <hr>
+      <h1>Signups</h1>
+      {# This is a link to allow manually navigating to the registration page, but clicking the button only copies the URl to the clipboard. #}
```

Comment:
`URl` isn't fully capitalized.

Also, how am I supposed to use this link to manually navigate to the page? I'd probably prefer a main link, followed by a smaller `copy link` button

Reply:
Right click, "open on new tab"

Seems like a small use case for a mentor to be at this event page and want to go to the sign up. But yeah the UX might be better if there was a smaller button to copy the link.

Reply:
I've separated these into a normal link as well as a `Copy 🔗` button that copies it to the clipboard.